autoencoder.py

```diff
@@ -0,0 +1,72 @@
+"""Implementation of feed forward autoencoder in TensorFlow 2.0"""
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+__version__ = '1.0.0'
+__author__ = 'Richard Ricardo'
+
+import tensorflow as tf
+from tensorflow.keras.layers import Dense
+from keras.models import Model
+
+class Autoencoder(tf.keras.Model):
```

Comment:
Where's the experiment using this class? @ChadElRicardo

Reply:
there's a second commit to this request **chore: add autoencoder notebook**  by the filename: 
 **feedfoward_autoencoder.ipynb**

Reply:
I didn't see it. Sorry. I'll review it today. Thank you.